CMakeLists.txt

```diff
@@ -816,6 +816,8 @@ if(NOT MSVC)
   if("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
     string(APPEND CMAKE_CXX_FLAGS " -Wno-range-loop-analysis")
     string(APPEND CMAKE_CXX_FLAGS " -Wno-pass-failed")
+    string(APPEND CMAKE_CXX_FLAGS " -Wmissing-prototypes")
+    string(APPEND CMAKE_CXX_FLAGS " -Wno-error=missing-prototypes")
```

Comment:
I assume this is temporary?

Reply:
We should use it permanently to check refactoring errors